_summaries/qiao2019micro-batch_2/dummy.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this file for???

Reply:
Github doesn't allow to create empty directory so added this dummy file